travis/samples/ngx-mariadb-oauth2-sass-infinispan/.yo-rc.json

```diff
@@ -1,12 +1,12 @@
 {
   "generator-jhipster": {
-    "baseName": "travisNg2SassNoi18n",
+    "baseName": "travisOauth2SassInfinispan",
     "packageName": "io.github.jhipster.travis",
     "packageFolder": "io/github/jhipster/travis",
     "serverPort": "8080",
     "authenticationType": "oauth2",
-    "cacheProvider": "ehcache",
-    "enableHibernateCache": true,
+    "cacheProvider": "infinispan",
+    "enableHibernateCache": false,
```

Comment:
This should be true else we are not testing much here

Reply:
done!